supabase/functions/_backend/utils/ip.ts

```
@@ -0,0 +1,75 @@
+    return value
+
+  const parts = value.split(':')
+  if (parts.length !== 2)
+    return value
+
+  const [high, low] = parts.map(part => Number.parseInt(part, 16))
+  if (
+    !Number.isFinite(high)
+    || !Number.isFinite(low)
+    || high < 0
+    || high > 0xffff
```

ESLint

[error] Invalid number literal casing.

(unicorn/number-literal-case)